sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/api/Azure.ResourceManager.AppConfiguration.netstandard2.0.cs

```diff
@@ -536,5 +589,27 @@ public static partial class ArmAppConfigurationModelFactory
         public static Azure.ResourceManager.AppConfiguration.DeletedAppConfigurationStoreData DeletedAppConfigurationStoreData(Azure.Core.ResourceIdentifier id = null, string name = null, Azure.Core.ResourceType resourceType = default(Azure.Core.ResourceType), Azure.ResourceManager.Models.SystemData systemData = null, Azure.Core.ResourceIdentifier configurationStoreId = null, Azure.Core.AzureLocation? location = default(Azure.Core.AzureLocation?), System.DateTimeOffset? deletedOn = default(System.DateTimeOffset?), System.DateTimeOffset? scheduledPurgeOn = default(System.DateTimeOffset?), System.Collections.Generic.IReadOnlyDictionary<string, string> tags = null, bool? isPurgeProtectionEnabled = default(bool?)) { throw null; }
+        public static Azure.ResourceManager.AppConfiguration.ReplicaData ReplicaData(Azure.Core.ResourceIdentifier id = null, string name = null, Azure.Core.ResourceType resourceType = default(Azure.Core.ResourceType), Azure.ResourceManager.Models.SystemData systemData = null, Azure.Core.AzureLocation? location = default(Azure.Core.AzureLocation?), string endpoint = null, Azure.ResourceManager.AppConfiguration.Models.ReplicaProvisioningState? provisioningState = default(Azure.ResourceManager.AppConfiguration.Models.ReplicaProvisioningState?)) { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ReplicaProvisioningState : System.IEquatable<Azure.ResourceManager.AppConfiguration.Models.ReplicaProvisioningState>
```

Comment:
Could we rename this to `AppConfigurationReplicaProvisioningState`?
We could do it using `rename-mapping`.

Reply:
Thanks! The `Replica: AppConfigurationReplica` under the `rename-mapping` section does not change `ReplicaProvisioningState `. 
So I added another line `ReplicaProvisioningState: AppConfigurationReplicaProvisioningState` and regenerated the code.